Server/Services/RatingService.cs

```diff
@@ -35,6 +35,19 @@ public RatingService(AppDbContext context, IWebHostEnvironment environment, IMem
                     .ToListAsync();
             });
 
+        public async Task<List<LeaderboardPlayer>> GetMapPlayersRating(List<ulong> players) =>
+            await _cache.GetOrCreateAsync(CacheKey, async entry =>
+            {
+                entry.AbsoluteExpirationRelativeToNow = _environment.IsProduction() ? TimeSpan.FromMinutes(5) : TimeSpan.FromTicks(1);
+                int rank = 1;
+                return _context.Players
+                    .OrderByDescending(p => p.Rating12v12)
+                    .AsEnumerable()
```

Comment:
FIY this would basically make the server fetch every single player row, so it should never be used like that